src/libraries/ExcelConverter/ParsedCellAnalyzer.cs

```diff
@@ -322,14 +322,14 @@ public override String Visit(VariadicOpNode node, Precedence context)
 // Might need to add error handling in case someone passes in wrong kind of Match object?
 public struct Range
 {
-    public Range()
-    {
```

Comment:
Just curious, why did you comment out this constructor here? Should I replicate this change in my testing/main branches too?

Reply:
oh good catch, the build was failing with an error saying that there couldn't be a constructor with empty parameters for a struct. It might have to do with downgrading the .net version (i'm not sure if that support is only in the newer version you were using for example).